src/test/java/org/jabref/gui/util/TextUtilTest.java

```diff
@@ -0,0 +1,150 @@
+        Assert.assertEquals("Regular", text.getFont().getStyle());
+        Assert.assertEquals(testText, text.getText());
+    }
+
+    @Test
+    public void testCreateTextBold() {
+        String testText = "this is a test text";
+        Text text = TextUtil.createText(testText, 12, TextUtil.TextType.BOLD);
+        Assert.assertEquals("Bold", text.getFont().getStyle());
+        Assert.assertEquals(testText, text.getText());
+    }
+
+    @Test
+    public void testCreateTextItalic() {
+        String testText = "this is a test text";
+        Text text = TextUtil.createText(testText, 12, TextUtil.TextType.ITALIC);
+        Assert.assertEquals("Italic", text.getFont().getStyle());
+        Assert.assertEquals(testText, text.getText());
+    }
+
+    @Test
+    public void testCreateTextMonospaced() {
+        String testText = "this is a test text";
+        Text text = TextUtil.createText(testText, 12, TextUtil.TextType.MONOSPACED);
+        Assert.assertEquals("Monospaced", text.getFont().getFamily());
+        Assert.assertEquals("Regular", text.getFont().getStyle());
+        Assert.assertEquals(testText, text.getText());
+    }
+
+    @Test
+    public void testTextToHTMLStringBold() {
+        String testText = "this is a test text";
+        Text text = TextUtil.createText(testText, 12, TextUtil.TextType.BOLD);
+        String htmlString = TextUtil.textToHTMLString(text);
+        Assert.assertEquals("<b>" + testText + "</b>", htmlString);
+    }
+
+    @Test
+    public void testTextToHTMLStringItalic() {
+        String testText = "this is a test text";
+        Text text = TextUtil.createText(testText, 12, TextUtil.TextType.ITALIC);
+        String htmlString = TextUtil.textToHTMLString(text);
+        Assert.assertEquals("<i>" + testText + "</i>", htmlString);
+    }
+
+    @Test
+    public void testTextToHTMLStringMonospaced() {
+        String testText = "this is a test text";
+        Text text = TextUtil.createText(testText, 12, TextUtil.TextType.MONOSPACED);
+        String htmlString = TextUtil.textToHTMLString(text);
+        Assert.assertEquals("<kbd>" + testText + "</kbd>", htmlString);
+    }
+
+    @Test
+    public void testTextToHTMLStringMonospacedBold() {
+        String testText = "this is a test text";
+        Text text = new Text(testText);
+        text.setFont(Font.font("Monospaced", FontWeight.BOLD, 12));
+        String htmlString = TextUtil.textToHTMLString(text);
+        Assert.assertEquals("<b><kbd>" + testText + "</kbd></b>", htmlString);
+    }
+
+    @Test
+    public void testTextToHTMLStringWithLinebreaks() {
+        String testText = "this\nis a\ntest text";
+        Text text = TextUtil.createText(testText, 12, TextUtil.TextType.NORMAL);
+        String htmlString = TextUtil.textToHTMLString(text);
+        Assert.assertEquals("this<br>is a<br>test text", htmlString);
+    }
+
+    private boolean checkIfTextsEqualsExpectedTexts(List<Text> texts, List<Text> expectedTexts) {
```

Comment:
Please extract this method to a new util class, since it is used in many tests.
Also remove the `System.out` statements.